backend/molgenis-emx2/src/main/java/org/molgenis/emx2/User.java

```diff
@@ -0,0 +1,13 @@
+package org.molgenis.emx2;
+
+public class User {
```

Comment:
maybe take a look at 'java records' for these types op value classes

Reply:
Ah yes, that is lovely. `public record User(String username) {}`
So far I had not looked at java 15 features that much. Do we want to go this way now?

Reply:
For now I add equals + hashcode